### ISSUE TITLE
Task not found: "precommit:app" when trying to use npm run all 

### ISSUE THREAD

**Issue by kelly-tock** [typicode/husky#466]
terminal output:
```
$ HUSKY_DEBUG=true git ca 'wip'
husky:debug pre-commit hook started...
husky > pre-commit (node v11.11.0)
ERROR: Task not found: "precommit:app", precommit:app2", precommit:app3"
husky > pre-commit hook failed (add --no-verify to bypass)
```

we have a mono repo that isn't set up with `lerna` or anything like that. 

at the root is this package.json(relevant parts)
```
{
  "name": "monoreponame",
  "dependencies": {
    "husky": "^1.3.1",
    "npm-run-all": "^4.1.5"
  },
  "scripts": {
    "precommit:app": "cd app && yarn precommit",
    "precommit:app2": "cd app2 && yarn precommit",
    "precommit:app3": "cd app3 && yarn precommit"
  },
  "husky": {
    "hooks": {
      "pre-commit": "npm-run-all precommit:app precommit:app2 precommit:app3"
    }
  }
}
```

and `app` etc are app folders in the mono repo. 

inside the `app` folder is

```
"precommit": "lint-staged",
```

it seems like if I run an install in a sub directory or sometimes seems random, the output starts happening. 

then if I `yarn add husky` from the root, then try the same commit, the scripts run as expected. 

I installed node via nvm.

**Comment by typicode** [comment]
It may be fixed in Husky v4. Feel free to re-open if not.

**Comment by zachsa** [comment]
I don't think this is fixed - I'm using Husky 4.2.3
